rome-modules/src/main/java/com/rometools/modules/mediarss/io/MediaModuleParser.java

```diff
@@ -152,7 +165,12 @@ public Module parse(final Element mmRoot, final Locale locale) {
 
                     try {
                         if (content.getAttributeValue("fileSize") != null) {
-                            mc.setFileSize(Long.valueOf(content.getAttributeValue("fileSize")));
+                            String fileSizeAttr = content.getAttributeValue("fileSize").replaceAll("\\s", "");
```

Comment:
Maybe handle spaces inside `parseFileSize`? For example, include them in the regex. Otherwise we're doing some parsing here and some parsing there.